pkg/k8s/service.go

```diff
@@ -414,6 +414,7 @@ func NewService(ip net.IP, externalIPs, loadBalancerIPs, loadBalancerSourceRange
 	loadBalancerSourceCIDRs := make(map[string]*cidr.CIDR, len(loadBalancerSourceRanges))
 
 	for _, cidrString := range loadBalancerSourceRanges {
+		cidrString = strings.TrimSpace(cidrString)
```

Comment:
This works, but why not do it `ParseService()`? It seems like `ParseService()` is the right level to do it at since we "validating" other fields there as well.